web/wp-content/themes/dctx-v2/inc/customizer/class-text-editor-custom-control.php

```diff
@@ -0,0 +1,79 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Class file names should be based on the class name with &quot;class-&quot; prepended. Expected class-text-editor-custom-control.php, but found phpcs-scan-dgNYpY.php (*WordPress.Files.FileName.InvalidClassFileName*).